01/app.js

```diff
@@ -1,2 +1,10 @@
 const randomNumber = Math.round(Math.random() * 20);
 console.log(randomNumber);
+
+if (randomNumber > 5) {
+    for (let i=5; i<=randomNumber; i++)
+    console.log(i);
```

Comment:
Przydałoby się nawiasy klamrowe. Dla jednej instrukcji nie trzeba, ale to dobry nawyk - jest czytelniej ;)